database/sql/V047__index_orders_by_quote_parameters.sql

```diff
@@ -0,0 +1,5 @@
+-- Orders that have the same `sell_token`, `buy_token` and `sell_amount` can have their `surplus_fee`
+-- updated based on the same underlying quote.
+-- To update an order's `surplus_fee` more quickly based on those parameters we need an index across
+-- all those columns.
+CREATE INDEX order_quoting_parameters ON orders USING BTREE (sell_token, buy_token, sell_amount);
```

Comment:
Is there a reason for the BTREE choice here? Wouldn't a `HASH` make more sense because we don't care about ordering?

Reply:
Yes, `HASH` would make more sense here. This was an oversight after copying and pasting another migration script. Good catch.
Fixed in [23b02b2](https://github.com/cowprotocol/services/pull/1037/commits/23b02b218f6fc128dd0dbd16a40d201c52bed320).